### ISSUE TITLE
False positive for `no-commented-out-code` with symbols other than latin

### ISSUE THREAD

**Issue by Arttse** [cartant/eslint-plugin-etc#28]
For example:
```ts
enum Data {
  // City
  Zero,
  // Город
  One,
  // 町
  Two,
  // İlçe
  Three,
  // Քաղաք
  Four,
}
```
All comments triggered, except `// City`

**Comment by cartant** [comment]
Should be fixed in 1.3.4.

**Comment by Arttse** [comment]
@cartant thank you for unexpectedly fast fix :sweat_smile: :+1: It works!

**Comment by cartant** [comment]
No worries. Anything to get rid of commented-out code; it's the worst. 🙂
